heart/test/ionicmodels/TestCodegenLong/TestCodegenLongAnalyticCvode.hpp

```diff
@@ -62,11 +62,19 @@ class TestCodegenLongAnalyticCvode : public CodegenLongHelperTestSuite
         args.push_back("--use-analytic-jacobian");
 
         // These have NaN in the jacobian due to massive exponentials
-        std::vector<std::string> bad_models = spectail_streatment_models(models, {"aslanidi_model_2009",
+        std::vector<std::string> bad_models = special_treatment_models (models, {"aslanidi_model_2009",
                                                                                   "hund_rudy_2004_a",
                                                                                   "livshitz_rudy_2007"});
         HeartConfig::Instance()->SetOdePdeAndPrintingTimeSteps(0.005, 0.1, 1.0);
+        // check test for negative concentrations works
+        TS_ASSERT_THROWS_ANYTHING(RunTests(dirname, {"negative_concentration_paci_hyttinen_aaltosetala_severi_ventricularVersion"}, args));
+
+        // initial value for membrane_L_type_calcium_current_f_gate in Shannon2004 is slightly above 1 (it is a probability)
+        std::vector<std::string> bigger_tolerance_models = special_treatment_models(models, {"Shannon2004"});
```

Comment:
We shouldn't need lines like this in the tests if codegen is no longer printing the `>1` check in the generated c++?